keel-core/src/main/kotlin/com/netflix/spinnaker/keel/lifecycle/LifecycleEventService.kt

```diff
@@ -0,0 +1,12 @@
+package com.netflix.spinnaker.keel.lifecycle
+
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+import org.springframework.stereotype.Component
+
+@Component
+class LifecycleEventService (
+  val lifecycleEventRepository: LifecycleEventRepository
+) {
+  fun getStepsForArtifactAndVersion(deliveryArtifact: DeliveryArtifact, version: String): List<LifecycleStep> =
+    lifecycleEventRepository.getSteps(deliveryArtifact, version)
+}
```

Comment:
This is just passing right through. Does this class add any value?

Reply:
Currently, it definitely does not. I thought that the pattern we wanted to use was that only "service" classes that abstract the repository get wired into the controllers / etc. So, I created this service. I can also remove it and just wire the repository into the `ApplicationService`. If I do that, and the logic gets more complicated, it's easy to put a service in place. 

I know @luispollo usually has opinions about this kind of thing. Thoughts?

Reply:
Yeah. I'd default to adding it if & when needed rather than adding a level of redirection before there's a good reason to

Reply:
Removing, discussed with @robfletcher offline. This just adds a layer of misdirection